outputs/gcpcloudrun.go

```diff
@@ -10,6 +10,10 @@ import (
 func (c *Client) CloudRunFunctionPost(falcopayload types.FalcoPayload) {
 	c.Stats.GCPCloudRun.Add(Total, 1)
 
+	if c.Config.GCP.CloudRun.JWT != "" {
+		c.AddHeader(AuthorizationHeaderKey, "Bearer "+c.Config.GCP.CloudRun.JWT)
+	}
```

Comment:
Compare to the following hunk:
https://github.com/falcosecurity/falcosidekick/blob/970a90504eb3d761c8cc9617a9153a9397ba5dbf/outputs/client.go#L181-L183